update-orphaning/index.js

```diff
@@ -1029,19 +1040,17 @@ function displayLoadErr() {
   var errDiv = document.getElementById("error-message");
   errDiv.textContent = "";
   var errB = document.createElement("b");
-  errB.textContent = "No data files found at the following location!";
+  errB.textContent = "Expected data file was not found at the following location!";
   errDiv.appendChild(errB);
   var errBR = document.createElement("br");
   errDiv.appendChild(errBR);
-  var errText = document.createTextNode("(URL: ");
+  var errText = document.createTextNode("URL: ");
   errDiv.appendChild(errText);
   var errA = document.createElement("a");
-  errA.setAttribute("href", DATA_URL + reportFile);
+  errA.setAttribute("href", DATA_URL + document.getElementById("weekly-dropdown").value);
   errA.setAttribute("target", "_blank");
-  errA.textContent = DATA_URL + reportFile;
+  errA.textContent = DATA_URL + document.getElementById("weekly-dropdown").value;
   errDiv.appendChild(errA);
-  errText = document.createTextNode(")");
-  errDiv.appendChild(errText);
```

Comment:
I just removed the parenthesis around the url. Now it is as follows
  var errText = document.createTextNode("URL: ");
  errDiv.appendChild(errText);
  var errA = document.createElement("a");
  errA.setAttribute("href", DATA_URL + document.getElementById("weekly-dropdown").value);
  errA.setAttribute("target", "_blank");
  errA.textContent = DATA_URL + document.getElementById("weekly-dropdown").value;
  errDiv.appendChild(errA);